File: java/UberBlack.java
import java.util.ArrayList;
import java.util.Map;

public class UberBlack extends Car {
    Map <String, Map<String, Integer>> typeCardAcepyrd; 
    ArrayList<String> setsMaterial;
    public UberBlack(String license, Account driver, Map <String, Map<String, Integer>> typeCardAcepyrd, ArrayList<String> setsMaterial) {
        super(license, driver);
        this.typeCardAcepyrd = typeCardAcepyrd;
        this.setsMaterial = setsMaterial;
        
    }
    
    void printData() {
        System.out.println(" license: " + license);
    }
}

File: PHP/UberBlack.php
<?php
require_once("Car.php");
class UberBlack extends Car {
    public $typeCardAcepted;
    public $material;

    function __construct($license, $driver, $typeCardAcepted, $material) {
        parent::__construct($license, $driver);
        $this->typeCardAcepted = $typeCardAcepted;
        $this->material = $material;
    }
}